### PR TITLE
Topic 1.2

### PR DESCRIPTION
**Pull request by nicolasronsmansglobant**

### CODE REVIEW

File: 01-HTML-CSS-Basics/Topic1-CSS/2.1CssBasics/index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        <meta charset="utf-8">
+        <meta http-equiv="x-ua-compatible" content="ie=edge">
+        <title>CSS BOOTCAMP</title>
+        <meta name="description" content="Esta es una pagina donde aprenderemos conceptos basicos sobre css">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <link rel="apple-touch-icon" href="apple-touch-icon.png">
+        <!-- Place favicon.ico in the root directory -->
+
+        <link rel="stylesheet" href="css/normalize.css">
```
Comment (nicolasronsmansglobant) [inline 01-HTML-CSS-Basics/Topic1-CSS/2.1CssBasics/index.html]: I can't find the file. Is this the case when normalize was breaking everything apart?

File: 01-HTML-CSS-Basics/Topic1-CSS/2.3.4y5/index.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        <link rel="stylesheet" href="css/estilo.css">
+      
+        <script src="js/vendor/modernizr-2.8.3.min.js"></script>
+</head>
+ 
+<body>
+    <!--[if lt IE 8]>
+            <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
+        <![endif]-->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/{{JQUERY_VERSION}}/jquery.min.js"></script>
```
Comment (nicolasronsmansglobant) [inline 01-HTML-CSS-Basics/Topic1-CSS/2.3.4y5/index.html]: {{JQUERY_VERSION}} should be replace by an existing version of jQuery.

File: 01-HTML-CSS-Basics/Topic1-CSS/2.1CssBasics/index.html (hunk 1 above)
Comment (angelestroisi) [inline 01-HTML-CSS-Basics/Topic1-CSS/2.1CssBasics/index.html]: ohh no!! this is that I say to day, I put my css before the normalize

Comment (angelestroisi) [inline 01-HTML-CSS-Basics/Topic1-CSS/2.1CssBasics/index.html]: * {
	border: 0;
	margin: 0;
	padding: 0;
}
body {
	background: #212121;
	color: #FFF;
	font-family: Helvetica, Tahoma;
	font-size: 1em;
	text-align: center;
}
 h1 { color: #2ECCFA; font-family: Verdana; 
 }

h2 {color: #5882FA; font-family: Tahoma;
}

File: 01-HTML-CSS-Basics/Topic1-CSS/2.3.4y5/index.html [hunk 3]
```diff
@@ -0,0 +1,74 @@
+!doctype html>
```
Comment (nicolasronsmansglobant) [inline 01-HTML-CSS-Basics/Topic1-CSS/2.3.4y5/index.html]: Whoops, small typo, the < missing here

File: 01-HTML-CSS-Basics/Topic1-CSS/2.3.4y5/index.html [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+            e.src='https://www.google-analytics.com/analytics.js';
+            r.parentNode.insertBefore(e,r)}(window,document,'script','ga'));
+            ga('create','UA-XXXXX-X','auto');ga('send','pageview');
+        </script>
+
+
+<header class="header">HEADER</header> 
+<nav class="navigation">
+     
+     <p><span>NAV</span></p>
```
Comment (nicolasronsmansglobant) [inline 01-HTML-CSS-Basics/Topic1-CSS/2.3.4y5/index.html]: Why the span here? :)

File: 01-HTML-CSS-Basics/Topic1-CSS/2.3.4y5/css/estilo.css [hunk 5]
```diff
@@ -0,0 +1,163 @@
+body {
+    background-color: #1C1C1C;
+}
+
+* {
```
Comment (nicolasronsmansglobant) [inline 01-HTML-CSS-Basics/Topic1-CSS/2.3.4y5/css/estilo.css]: Try to avoid assigning rule with the * selector. These rules can be set for the html/body.

File: 01-HTML-CSS-Basics/Topic1-CSS/2.3.4y5/css/estilo.css [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+   font-size: 20px;
+   font-family: Verdana;   
+   font-weight:bold;
+
+
+}
+
+nav,aside,header,footer {
```
Comment (nicolasronsmansglobant) [inline 01-HTML-CSS-Basics/Topic1-CSS/2.3.4y5/css/estilo.css]: One selector per line would be better.

File: 01-HTML-CSS-Basics/Topic1-CSS/2.3.4y5/index.html (hunk 3 above)
Comment (angelestroisi) [inline 01-HTML-CSS-Basics/Topic1-CSS/2.3.4y5/index.html]: Yes the start of DOCTYPE

Comment (angelestroisi) [inline 01-HTML-CSS-Basics/Topic1-CSS/2.3.4y5/index.html]: I just say you in the meeting

**Comment by nicolasronsmansglobant** [comment]
Good work! A bit messy sometimes but nothing too important. Try to focus on writing down more code from your own, and also applying good practices about file-naming and code cleanness.
